public/glossary.json

```diff
@@ -2,21 +2,27 @@
   "glossary": [
     {
       "term": "annuity",
-      "description": "A type of retirement income that provides you with a regular payment, usually for life."
+      "description": "A retirement income product that guarantees a regular income for the rest of your life."
     },{
-      "term": "cash lump sum",
-      "description": "An amount of cash set by law that you can take at retirement free of tax. It's usually up to a quarter of your pension. Sometimes simply referred to as 'tax-free cash' or 'cash lump sum'."
+      "term": "lifetime annuity",
+      "description": "A retirement income product that guarantees a regular income for the rest of your life."
```

Comment:
It does have the same meaning. per: https://moneyadviceservice.tpondemand.com/RestUI/Board.aspx?acid=5D6AC326B4D272C23E446C28586BAF51#page=board/5624876525867731357&appConfig=eyJhY2lkIjoiREI4ODcwRjkxMDNDRTM2NTlBMzhDNTRBRTVBNUU1N0UifQ==&boardPopup=userstory/8154/silent